game/external/discord_jak2.cpp

```diff
@@ -0,0 +1,84 @@
+#include "discord_jak2.h"
+
+namespace jak2 {
+const std::map<std::string, std::string> level_names = {
+    {"intro", "Intro"},
+    {"title", "Title screen"},
+    {"prison", "Prison"},
+    {"forexit", "Fortress (Escape)"},
+    {"fordump", "Fortress (Ammo Dump)"},
+    {"forresc", "Fortress (Rescue Friends)"},
+    {"ctyslum", "Haven City (Slums)"},
+    {"ctyport", "Haven City (Port)"},
+    {"ctyfarm", "Haven City (Gardens)"},
+    {"ctyind", "Haven City (Industrial Zone)"},
+    {"ctymark", "Haven City (Bazaar)"},
+    {"ctypal", "Haven City (Palace Area)"},
+    {"ctygen", "Haven City (Main Town)"},
+    {"stadium", "Stadium"},
+    {"stadiumb", "Stadium (Class 3 Race)"},
+    {"stadiumc", "Stadium (Class 2 Race)"},
+    {"stadiumd", "Stadium (Class 1 Race)"},
+    {"skatea", "Stadium (JET-Board Course)"},
+    {"palshaft", "Palace Lobby"},
+    {"palcab", "Palace Cable"},
+    {"palroof", "Palace Roof"},
+    {"throne", "Throne Room"},
+    {"palent", "Palace Entrance"},
```

Comment:
`palent` is the level that has every part of the Palace interior that's not the shafts, roof or throne room itself. Maybe "Palace Interior" is better.